scalafix-core/src/main/scala/scalafix/config/FilterMatcher.scala

```diff
@@ -11,6 +11,7 @@ case class FilterMatcher(
     @ExtraName("include") includeFilters: Regex,
     @ExtraName("exclude") excludeFilters: Regex
 ) {
+  def matches(path: AbsolutePath): Boolean = matches(path.toString())
```

Comment:
Good question. This matches against the absolute path but I think --include/--exclude should match against the relative path. I updated the PR.